lib/wiki_course_edits.rb

```diff
@@ -70,6 +70,25 @@ def enroll_in_course(enrolling_user:)
     add_template_to_sandbox
   end
 
+  # Removes existing template from the disenrolling student's userpage, and
+  # also from their talk page
+  def disenroll_from_course(disenrolling_user:)
+    generator = WikiUserpageOutput.new(@course)
+
+    @disenrolling_user = disenrolling_user
+    # Remove existing template from the user page
+    template = generator.enrollment_template
+    user_page = "User:#{@disenrolling_user.username}"
+    summary = generator.disenrollment_summary
+    remove_content_from_page(user_page, template, summary)
+
+    # Remove existing template from the user's talk page
+    talk_template = generator.enrollment_talk_template
+    talk_page = "User_talk:#{disenrolling_user.username}"
+    talk_summary = "removing {{#{template_name(@templates, 'user_talk')}}}"
+    remove_content_from_page(talk_page, talk_template, talk_summary)
+  end
```

Comment:
Sanity check: should I do something with the template in sandbox to revert the `add_template_to_sandbox` action?

Reply:
I don't think that's necessary, as that's not a place other editors will look to understand the context for the un-enrolled student's edits.

Reply:
I think I edited the message adding a second question after you answered, so just checking you saw both questions.

Reply:
Ah, I missed the second question. I think yes, checking `enrollment_edits_enabled?` for this action makes sense.

Reply:
Added the check and spec for that on 00cf010f45cfc3db2c60db59fc01c3e63189de8f